ci-operator/config/openshift/cluster-samples-operator/openshift-cluster-samples-operator-master__okd.yaml

```diff
@@ -0,0 +1,52 @@
+base_images:
+  base:
+    name: "4.6"
+    namespace: origin
+    tag: base
+  ocp_builder_rhel-7-golang-1.13:
+    name: builder
+    namespace: ocp
+    tag: rhel-7-golang-1.13
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+images:
+- dockerfile_path: Dockerfile.okd
+  from: base
+  inputs:
+    ocp_builder_rhel-7-golang-1.13:
+      as:
+      - registry.svc.ci.openshift.org/ocp/builder:rhel-7-golang-1.13
+  to: cluster-samples-operator
+promotion:
+  name: "4.6"
+  namespace: origin
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.6"
+  namespace: origin
+tests:
+- as: unit
+  commands: make test-unit
+  container:
+    from: src
+- as: e2e-aws
+  steps:
+    cluster_profile: aws
+    workflow: openshift-e2e-aws
+- as: e2e-aws-upgrade
```

Comment:
I would expect image ecosystem e2e minimally to be included to actually validate sample imagestreams/templates

Did you try to run it against a cluster with OKD samples?

Reply:
Oh, right, adding that test

Didn't try the actual cluster just yet

Reply:
ok ... the operator e2e, jenkins e2e, and build e2e would also fall into this camp

Reply:
Thanks, updated. Not sure about `e2e-operator` - does it use any imagestreams?

Reply:
it makes sure they and templates are installed ... still valid checks for okd imo